CHANGES.md

```diff
@@ -517,7 +517,7 @@ OpenSSL 3.2
    *Paul Dale*
 
  * Subject or issuer names in X.509 objects are now displayed as UTF-8 strings
-   by default.
+   by default. Also spaces surrounding `=` in Subject output are removed.
```

Comment:
Is this only for Subject, or also for Issuer?

Reply:
I think in all DNs

Reply:
Ok...in that case the wording should be adjusted accordingly

Reply:
Done